### DISCUSSION TITLE
Set tabs option to current window only

### DISCUSSION THREAD

**Discussion by miffi** [qutebrowser/qutebrowser#7497] [Q&A] [answered]
I use the vertical tab bar. Sometimes I want to remove the bar in a window to tile it with another program. For that I have this bind:
```py
config.bind(',t', 'config-cycle tabs.show never always')
```
However, this changes the tab visibility for all the windows.

Is there any way for me to change the tab visibility for one window only?

**Comment by The-Compiler** [comment, marked as answer]
Nope, see:

- #2314
